### ISSUE TITLE
(User feedback) iOS VoiceOver: Unable to move multiple face-up cards simultaneously between dealt card piles

### ISSUE THREAD

**Issue by gbarkerz** [gbarkerz/AccessibleSolitaire#51]
This was with Accessible Solitaire iOS Version 2.7.

The feedback stated that the player used to be able to move multiple face-up cards simultaneously from one dealt card pile to another while VoiceOver is running, but this no longer works. For example, they had a jack of diamonds in one pile, and wanted to move both the 10 of spades and the 9 of hearts to that pile. But when they double-tapped the pile with the 10 of spades and 9 of hearts, only the 9 of hearts was selected. So that when they went to move what I thought was both cards to where the jack of diamonds was, the "unsuccessful" sound played. They checked the app setting, and everything was set the way they liked it.

**Comment by gbarkerz** [comment]
This has been fixed in iOS Version 2.9, and can no longer be reproduced by the user who originally raised the issue.
